src/js/features/channel-state.js

```diff
@@ -135,10 +135,10 @@ module.exports = function(event) {
             var msg = event.tags['msg-id'];
 
             if (msg === 'msg_slowmode' || msg === 'msg_timedout') {
-                var matches = /([0-9]+)/.exec(event.message);
+                var matches = /\s([0-9]+)/.exec(event.message);
                 if (!matches) return;
 
-                var seconds = parseInt(matches[1], 10);
+                var seconds = parseInt(matches[0], 10);
```

Comment:
@night No reason why it changed. I just modified it by mistake. Should I fix it?